inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -253,15 +253,13 @@ apt install libsodium-dev cmake
 
 ##  Importing the data
 
-To illustrate the different analyses, we use real data reporting daily numbers
-of COVID-19 hospitalisations in England as of the 24 October 2020, broken down
-to the hospital and National Health Service (NHS) region level. The data is
-available online from the NHS England's
-[website](https://www.england.nhs.uk/statistics/statistical-work-areas/covid-19-hospital-activity/).
-The dataset analysed here is a simplified version, providing incidence of
-hospital admissions by NHS trust.
+To illustrate the different analyses, we use real data reporting line list 
+(individual level) data of Covid-19 cases in England in the second half of 2020.
+The data was downloaded from <global.health>:
 
-The data file is named "*covid_hosp_uk_20201024.xlsx*" and is located in
+> Xu, B., Gutierrez, B., Mekaru, S. et al. Epidemiological data from the COVID-19 outbreak, real-time case information. Sci Data 7, 106 (2020). https://doi.org/10.1038/s41597-020-0448-0 
```

Comment:
I feel like this is conflicting with the changes that I'm making in the Tx_pipeline_update anyway, but it's okay to leave as an example

Reply:
Good call. Feel free to remove it later down the line if it no longer matches with the content of the rest of the report.